swap/chain/backend.go

```diff
@@ -2,7 +2,6 @@ package chain
 
```

Comment:
What is the rationale behind `chain`?

Reply:
Primarily because I could not come up with something better. The code in the package is for generic block`chain` related stuff.